utils/src/main/kotlin/Constants.kt

```diff
@@ -28,6 +28,11 @@ const val ORT_FULL_NAME = "OSS Review Toolkit"
  */
 const val ORT_DATA_DIR_ENV_NAME = "ORT_DATA_DIR"
 
+/**
+ * The name of the ORT main configuration file.
+ */
+const val ORT_MAIN_CONFIG_FILENAME = "ort.conf"
```

Comment:
How about just `ORT_CONFIG_FILENAME` ?

Reply:
Sorry for not indicating it in the first place, this was just meant as (optional) suggestion.

Reply:
After giving it some though again I decided to drop the `MAIN`.